### DISCUSSION TITLE
[Feature?] Debugging Info via WebSerial

### DISCUSSION THREAD

**Discussion by b-a-9-0** [tbnobody/OpenDTU#159] [Ideas]
I don´t know if this makes any sense, but I think in some case it would be very helpful to print the debug messages on WebSerial (e. g. https://github.com/ayushsharma82/WebSerial) instead of hardware serial. Most users don´t always have the ESP32 attached to a PC. 

I hacked the lib mentioned above into my local branch and it simply works!

**Comment by timguy** [comment]
I would also prefer this way.

Maybe you could open a pull request?
Or could you publish your fork to spread this?
